src/thor/dijkstras.cc

```diff
@@ -214,6 +214,9 @@ void Dijkstras::ExpandInner(baldr::GraphReader& graphreader,
                 costing_->EdgeCost(directededge, tile, offset_time.second_of_week, flow_sources) +
                 transition_cost;
     } else {
+      if (opp_pred_edge == nullptr) {
+        continue;
+      }
```

Comment:
there is no point to this, you protected against it on line 361 below

```suggestion
```

Reply:
@kevinkreiser I think on line 361 we can pass `opp_pred_edge` as `nullptr`. Or am I wrong?
```
const baldr::DirectedEdge* opp_pred_edge = nullptr;
if (expansion_direction == ExpansionType::reverse) {
  const auto opp_edge_tile = graphreader.GetGraphTile(pred.opp_edgeid());
  if (opp_edge_tile != nullptr) {
    opp_pred_edge = opp_edge_tile->directededge(pred.opp_edgeid());
  }
}
```

Reply:
sorry im saying on line 361 you should not let it get to this point. basically you may as well stop it sooner than let it get to here

Reply:
Thanks for the explanation :)